### PR TITLE
YLP-1846 migrate bg bar widget

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/dumb/src/common/stats/cbg-utils.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import styles from './cbg-colors.css'
+
+export const computeCBGStyle = (value: number): { backgroundColor: string, color: string, left: string } => {
```
Comment (fatb38) [inline packages/dumb/src/common/stats/cbg-utils.tsx:30]: Interface for return please 😁

File: packages/dumb/src/common/tooltips/stat-tooltip.tsx [hunk 2]
```diff
@@ -27,74 +27,34 @@
 
 import React, { FunctionComponent } from 'react'
 import ReactMarkdown from 'react-markdown'
-
-import colors from '../../styles/colors.css'
 import styles from './stat-tooltip.css'
-import Tooltip from './tooltip'
+import { Tooltip, withStyles } from '@material-ui/core'
```
Comment (fatb38) [inline packages/dumb/src/common/tooltips/stat-tooltip.tsx:31]: Default import instead for MUI components : `import Tooltip from '@material-ui/core/Tooltip'`

File: packages/dumb/src/index.ts [hunk 3]
```diff
@@ -25,13 +25,21 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { CBGPercentageBarChartMemoized as CBGPercentageBarChart } from './common/stats/cbg-percentage-bar-chart'
+import { CBGMeanStatMemoized as CBGMeanStat } from './common/stats/cbg-mean/cbg-mean-stat'
+import {
+  CBGPercentageBarChartMemoized as CBGPercentageBarChart
```
Comment (fatb38) [inline packages/dumb/src/index.ts:30]: this is nice 😉👍🏻

File: packages/yourloops/test/integration/assert/trends.ts [hunk 4]
```diff
@@ -48,14 +50,14 @@ export const checkTrendsTidelineContainerTooltips = async () => {
...
-export const checkTrendsStatsWidgetsTooltips = () => {
+export const checkTrendsStatsWidgetsTooltips = async () => {
   const statsWidgets = within(screen.getByTestId('stats-widgets'))
-  checkStatTooltip(statsWidgets, 'Avg. Daily Time In Range', TIME_IN_RANGE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Avg. Glucose (CGM)', AVG_GLUCOSE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Sensor Usage', SENSOR_USAGE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'GMI (estimated HbA1c)', GMI_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Standard Deviation', STANDARD_DEVIATION_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'CV (CGM)', CV_TOOLTIP)
+  await checkStatTooltip(statsWidgets, 'Avg. Daily Time In Range', TIME_IN_RANGE_TOOLTIP)
```
Comment (fatb38) [inline packages/yourloops/test/integration/assert/trends.ts:55]: await is due to MUI tooltip right ? waiting for it to disapear ?

**Review by fatb38 — CHANGES_REQUESTED** [review]

File: packages/dumb/package.json [hunk 5]
```diff
@@ -35,7 +35,7 @@
     "lint": "eslint .",
     "lint-fix": "npm run lint -- --fix",
     "postbuild": "npm run copy-files",
-    "copy-files": "copyfiles src/**/*.css src/**/**/*.css src/**/*.svg dist/"
+    "copy-files": "copyfiles src/**/*.css src/**/**/*.css src/**/**/**/*.css src/**/*.svg dist/"
```
[the remark this answers is not in the document]
Comment (TimCanu) [reply packages/dumb/package.json:38]: I don't know... If you know how to make this cleaner please help me

File: packages/yourloops/test/integration/assert/trends.ts (hunk 4 above)
Comment (TimCanu) [reply packages/yourloops/test/integration/assert/trends.ts:55]: yes

File: packages/dumb/src/common/stats/cbg-utils.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (TimCanu) [reply packages/dumb/src/common/stats/cbg-utils.tsx:30]: As discuss, fuck switch cases

**Comment by fatb38** [comment]
Don't forget to rename the folder `common` into `components` before merge 😉

**Review by fatb38 — APPROVED** [review]
Don't forget to move settings folder in components and it's ok for me !

**Pull request merged**